src/components/molecules/branchCard/BranchCard.module.scss

```diff
@@ -0,0 +1,76 @@
+@import "../../../foundations/sass/all";
+
+.branch-card {
+  align-items: center;
+  background-color: $branch-card-background-color;
```

Comment:
We don't need background color here, because we can place it in the footer layout. It would be easy to control if we added a background color only to BranchCard 's container. For storybook, we can add background using decorators. Please remove `background-color` here and add it storybook